OutOfSchool/Tests/OutOfSchool.WebApi.IntegrationTests/ProviderServiceIntergrationTests/ProviderServiceUpdate.cs

```diff
@@ -0,0 +1,241 @@
+namespace OutOfSchool.WebApi.IntegrationTests.ProviderServiceIntergrationTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using FluentAssertions.Common;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.Extensions.Localization;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using NUnit.Framework;
+    using OutOfSchool.Services;
+    using OutOfSchool.Services.Models;
+    using OutOfSchool.Services.Repository;
+    using OutOfSchool.Tests;
+    using OutOfSchool.Tests.Common.TestDataGenerators;
+    using OutOfSchool.WebApi.Models;
+    using OutOfSchool.WebApi.Services;
+    using OutOfSchool.WebApi.Util;
+
+    [TestFixture]
+    public class ProviderServiceUpdate
+    {
+        private const string FakeUserId = "cqQQ876a-BBfb-4e9e-9c78-a0880286ae3c";
+
+        private IProviderService providerService;
+
+        // private Mock<IProviderRepository> providersRepositoryMock;
+        private Mock<IEntityRepository<User>> usersRepositoryMock;
```

Comment:
It seems all this mock are used only inside SetUp and are neer reused anywhere. Why are they put into class scope - shouldn't they be just SetUp method local variables?